allennlp/modules/text_field_embedders/basic_text_field_embedder.py

```diff
@@ -0,0 +1,54 @@
+from typing import Dict
+
+from overrides import overrides
+import torch
+
+from allennlp.common import Params
+from allennlp.common.checks import ConfigurationError
+from allennlp.data import Vocabulary
+from allennlp.experiments import Registry
+from allennlp.modules import TextFieldEmbedder, TokenEmbedder
+
+
+@Registry.register_text_field_embedder("basic")
+class BasicTextFieldEmbedder(TextFieldEmbedder):
```

Comment:
I answered this in person, but just in case anyone else stumbles upon this conversation, the second `TextFieldEmbedder` that I have in mind is one that loads a pre-trained model, runs it on some data, and pulls out some intermediate output for us as token embeddings for a separate model, similar to what's done in many current multi-task / semi-supervised NLP models.  That implementation is coming soon, after we've implemented some more concrete models to use it with.

I don't have any other good ideas for what this might look like, so I'd imagine there are just those two subclasses in here.